src/OpenLoco/Localisation/StringManager.h

```diff
@@ -53,14 +56,34 @@ namespace OpenLoco
         constexpr uint8_t window_colour_1 = 13;
         constexpr uint8_t window_colour_2 = 14;
         constexpr uint8_t window_colour_3 = 15;
-        constexpr uint8_t window_colour_4 = 16;
+        constexpr uint8_t window_colour_4 = 16; // Not used
 
-        // Arguments: int8_t, int8_t
+        // Arguments (2 bytes): int8_t, int8_t
         constexpr uint8_t newline_x_y = 17;
+        // 18 Not used
+        // 19 Not used
+        // 20 Not used
+        // 21 Not used
+        // 22 Not used
```

Comment:
Should we just group these, e.g.:
```suggestion
        // 18–22: not used
```